### DISCUSSION TITLE
Twingo III EasyLink Rear Camera Issues

### DISCUSSION THREAD

**Discussion by sasio141** [cedricp/ddt4all#1278] [Q&A]
Hi.
I add EasyLink to my 2017 Twingo III PH1 based on Visteon.

I add radio from Clio V, multimedia gate/screen/cover/button from Twingo III and made auto configuration.

I have issue with rear camera : when I turn reverse gear and car not move is everything okay, but when I start reversing the car, view from camera turn off (black screen), and correct working goes on after stop a car. 

Maybe is some parameter to change in DDt4All to change ? In my Clio IV for example when I start reversing faster than 10km/h visualisation of parking sensor (HMI) turn off..

https://github.com/user-attachments/assets/3180d0a0-aba5-4ca4-9c90-64208b054f4a

Any ideas?

**Comment by AndyIreland** [comment]
have you checked in this area of restrictions to see if make a difference?:

![image](https://github.com/user-attachments/assets/fec895a6-9702-49e4-a619-942d490f927b)

**Reply by sasio141** [reply]
I must check.
We add Clio V radio to Twingo III cause are wireless CarPlay. Albo add navigation and after update software navigation not working (nothing do after try open Navigation button).

Restriction video i Think is removed cause my friend told me about that.

In this 3 position, absent is correct value or not?

**Reply by AndyIreland** [reply]
yes i think absent is correct value to unlock. because if 'present' that would mean restriction is present

**Reply by sasio141** [reply]
Problem is only when can move back. When I turn on reverse gear and not move is okay, but when I reversing faster than 0 km/h is black screen :(

**Reply by maldenis** [reply]
Hello! Did you manage to solve your problem somehow? I also want to install additional all-round view cameras (MVC) that were installed from the factory. There is a problem with changing the Easylink parameters in the Camera config tab (ddt4all). The new values are not saved.

**Comment by sasio141** [comment]
Hi, problem is with unlock gateway in your car. Any changed must be after unlock the gateway.Wysłane z iPhone'aWiadomość napisana przez maldenis ***@***.***> w dniu 6 gru 2024, o godz. 12:53:﻿
Hello! Did you manage to solve your problem somehow? I also want to install additional all-round view cameras (MVC) that were installed from the factory. There is a problem with changing the Easylink parameters in the Camera config tab (ddt4all). The new values are not saved.

**Reply by maldenis** [reply]
I connected the CAN bus directly to Easylink (bypassing the gateway), but still there is no change in these settings.

**Reply by AndyIreland** [reply]
> I connected the CAN bus directly to Easylink (bypassing the gateway), but still there is no change in these settings.

on twingo 3 it says the touchscreen is mfd , not easylink. 
anyway, did you make sure to press this button after changes?
![image](https://github.com/user-attachments/assets/b2db6012-c096-4e65-aec8-9817a8d9e5a8)

**Comment by sasio141** [comment]
Hi in my twingo is easy link from clio V. Any changes i confirm by button Key off on left top site. At monday i will check this parameters for youWysłane z iPhone'aWiadomość napisana przez Andy Robus ***@***.***> w dniu 7 gru 2024, o godz. 16:31:﻿

I connected the CAN bus directly to Easylink (bypassing the gateway), but still there is no change in these settings.

on twingo 3 it says the touchscreen is mfd , not easylink.
anyway, did you make sure to press this button after changes?
image.png (view on web)

**Reply by franzkrk** [reply]
Hey did you ever fogure it out? I have same issue on my clio 5 because i did an update for the firmware. I have the old parameters saved maybe we can help each other?

**Comment by sasio141** [comment]
Hi. Just changed a radio for new one Wysłane z iPhone'aWiadomość napisana przez franzkrk ***@***.***> w dniu 9 cze 2025, o godz. 14:37:﻿
Hey did you ever fogure it out? I have same issue on my clio 5 because i did an update for the firmware. I have the old parameters saved maybe we can help each other?

**Reply by franzkrk** [reply]
Ah so you never looked in the parameters of the DDT4All? :/

**Comment by sasio141** [comment]
I Just looked but everything looked okay. I was update software, reprogrammed for confirm VIN number but without any resoult.Wysłane z iPhone'aWiadomość napisana przez franzkrk ***@***.***> w dniu 9 cze 2025, o godz. 19:28:﻿
Ah so you never looked in the parameters of the DDT4All? :/

**Reply by franzkrk** [reply]
Shit that means i probalbly wont have luck. Anyway i will look on ddt4all there are a few camera parameter settings. Mine stops working after driving more than 4km backwards was it same for you?

**Comment by sasio141** [comment]
I had this same before radio changedWysłane z iPhone'aWiadomość napisana przez franzkrk ***@***.***> w dniu 10 cze 2025, o godz. 09:42:﻿
Shit that means i probalbly wont have luck. Anyway i will look on ddt4all there are a few camera parameter settings. Mine stops working after driving more than 4km backwards was it same for you?

**Reply by franzkrk** [reply]
Changing parameters in camera settings in ddt4all worked  it was set to 0kmph and i had to turn it to 10kmph.

**Reply by SANTOSADEMAR** [reply]
> Changing parameters in camera settings in ddt4all worked it was set to 0kmph and i had to turn it to 10kmph.

I'm using ECU-Tweaker for Android but I can't find the module to change the km/h. Can anyone help me?

**Reply by SANTOSADEMAR** [reply]
I found

Navigation-UCC-ITM / Configuration / Vehicle Information $DAB0 / Camera paramentros / Vehicle Speed "Too High"
